EOLib/Domain/Account/AccountActions.cs

```diff
@@ -71,7 +71,7 @@ public async Task<AccountReply> CheckAccountNameWithServer(string accountName)
             var reply = (AccountReply)response.ReadShort();
             if (reply >= AccountReply.OK_CodeRange)
             {
-                _playerInfoRepository.AccountCreateID = (ushort)reply;
```

Comment:
I would rather have these properties (and associated parameters) named after which session ID they are rather than naming them all `SessionID`. I think it hurts readability, especially when there are multiple competing different types of session IDs, and multiple different properties across repositories for tracking said session IDs. I'd be ok with calling it `AccountCreateSessionID` if you really want to update the naming.

Reply:
I guess a name like ‘PregameSessionID’ might be better? It’s used for account create, character create, character delete, welcome agree, and welcome message packets.

I just called them all session_id in reoserv because only one could ever be in use at a time 🤷‍♂️

Reply:
Alright, after checking things out a little more thoroughly here's where I've landed.

Character create/delete IDs aren't persisted anywhere; they're just used within the same controller method. Account create should be refactored to work the same way - there's really no need to have this property in a repository.

That means that `PlayerInfoRepository` should have a single ID - `PlayerID` - since that's what this actually is. I guess it could be called `SessionID` but I've been calling it `PlayerID` for 8+ years so I'd prefer to keep it with that name. This ID should be set during welcome.

Here's what I'd like for this PR:
1. Remove setting the session ID during network connection initialization, since it isn't needed anywhere else beyond completing the handshake
2. Keep the name of `PlayerID` instead of `SessionID`
3. Optional: refactor account create to return the account create ID from the call to `AccountActions.CheckAccountNameWithServer` in `AccountController`, following a similar pattern as character create/delete where the session ID is passed as a parameter to `AccountActions.CreateAccount`.
    - If you don't want to take this on, keep AccountCreateID as a separate property for now and I can refactor it later

Reply:
I've applied your suggestions :)